README.md

```diff
@@ -4,13 +4,13 @@
 
 The GC Digital Talent app is divided into multiple services, each treated as its own sub-project:
 
-- `/api`, the API service
-- `/apps`, (new) location for frontend applications
+- `/api`, API service
+- `/apps`, frontend applications
 - `/packages`, npm packages used within `/apps`
 - `/apps/e2e`, e2e testing with [`Cypress`](/apps/e2e/cypress/README.md)
-- `/tc-report`, containing static content generated with Jekyll, mostly the Talent Cloud report. This used to be maintained in a [separate repo](https://github.com/GCTC-NTGC/tc-report), but now the entire project has been imported into this repo. See [`/tc-report/README.md`](/tc-report/README.md) for instructions on how to run Jekyll and re-generate the site content.
+- `/tc-report`, static content, mostly the Talent Cloud report, generated with [`Jekyll`](/tc-report/README.md)
```

Comment:
It feels a little funny to have a link with the text "Jekyll" go to our tc report readme.  I'm not sure what the link text should be.  "Read more"?

Reply:
Ya, I was following the convention that `Cypress` started, but am open to removing both! Thoughts?

Reply:
Yeah, the Cypress page is actually about Cypress though  (despite the H1 title on the page  :laughing: ) so I think that one's OK.  

Maybe the directory name could be the link?  
```
[`/tc-report`](/tc-report/README.md), static content, mostly the Talent Cloud report, generated with Jekyll
```

Reply:
Please see a85d688962eb27268d100b63b6d866a9e99d0954.